docs/src/.overrides/assets/stylesheets/extra.css

```diff
@@ -91,3 +91,7 @@ html a[title="YouTube"].md-social__link svg {
     /* previous/next text */
     /* --md-footer-fg-color: var(--dj-white); */
 }
+
+[data-md-color-scheme="slate"] .jupyter-wrapper .Table td {
+    color: var(--dj-primary);
```

Comment:
Do we perhaps want this to be dark background with light text?

Reply:
Yes - we can certainly do that. Here's an example. We can go with any background color and text color combination now but I've picked `dj-black` and `dj-primary` in the example below. If that looks better, I can push a commit with the update. 

![image](https://user-images.githubusercontent.com/52367253/220472944-f523cdf5-ed8f-4594-825b-08ad8efbd250.png)

Reply:
I worry that this is overuse of our blue. I think I'd flip what you have - Blue on black for the header (using blue or bold for primary keys), black on grey for the contents.

Reply:
I'd suggest a dev branch on your fork if you want to trying things out as committed. Maybe you're doing this already, but, you'll need to rerun the notebook in question to refresh any info coming from `preview.py`. CSS does have some tools for prioritizing one rule over another

Lmk if you want to schedule a co-coding session to work on the table header

Reply:
Ah, okay. I was not running the notebook again. One reservation of using `preview.py` for this is that I don't believe `preview.py` makes any distinction between light and dark mode at this point. It's probably not much more effort to include it but it adds time for implementing and testing. So before I dive into it, I want to ensure we'll be using this approach on the site for future tutorials as well? If so, I'm happy to proceed with developing it on a dev branch and potentially scheduling a co-coding session.

Reply:
How about we set the text color of the table to be `--dj-black`? Think that might be cleaner and more consistent.